department-of-veterans-affairs/caseflow
# EP Claim Label Add success alert and redirect to Veteran Case Search after edit

Resolves [CASEFLOW-365](https://vajira.max.gov/browse/CASEFLOW-365)

### Description
Add redirect to **Veteran Case Search** page after EP Claim label edit and show success alert

### Acceptance Criteria
- [ ] Page redirects to **Veteran Case Search**
- [ ] Success alert appears at top of search page 

### Testing Plan
1. Search for to a supplemental claim or higher level review that has been established
2. Click **Review Type** link
3. Click **Edit Claim Label**
4. Submit edit
5. Verify that page redirects to **Veteran Case Search** and success alert appears on page

### User Facing Changes

 BEFORE|AFTER
 ---|---
<img width="1155" alt="Screen Shot 2021-03-25 at 6 37 34 PM" src="https://user-images.githubusercontent.com/68879427/112564862-40b44200-8d99-11eb-8cc0-f3605890fb90.png">|<img width="1083" alt="Screen Shot 2021-03-31 at 10 44 28 AM" src="https://user-images.githubusercontent.com/68879427/113188321-bfccce80-920e-11eb-9f7d-7a00c8a44de7.png">
